contracts/payloads/IGP41/PayloadIGP41.sol

```diff
@@ -0,0 +1,953 @@
+                expandDuration: (userBorrowData_ >>
+                    LiquiditySlotsLink.BITS_USER_BORROW_EXPAND_DURATION) & X24,
+                baseDebtCeiling: getRawAmount(token_, 0, 7_500_000, false), // $7,500,000
+                maxDebtCeiling: BigMathMinified.fromBigNumber(
+                (userBorrowData_ >>
+                    LiquiditySlotsLink.BITS_USER_BORROW_MAX_BORROW_LIMIT) & X18,
+                DEFAULT_EXPONENT_SIZE,
+                DEFAULT_EXPONENT_MASK
+            )
+            });
+
+            LIQUIDITY.updateUserBorrowConfigs(config_);
+        }
+    }
+
+    function getVaultConfig(
+        address vault_
+    ) internal view returns (VaultConfig memory configs_) {
+        uint vaultVariables2_ = IFluidVaultT1(vault_).readFromStorage(bytes32(uint256(1)));
+        configs_.supplyRateMagnifier = uint16(vaultVariables2_ & X16);
+        configs_.borrowRateMagnifier = uint16((vaultVariables2_ >> 16) & X16);
+        configs_.collateralFactor = (uint16((vaultVariables2_ >> 32) & X10)) * 10;
+        configs_.liquidationThreshold = (uint16((vaultVariables2_ >> 42) & X10)) * 10;
+        configs_.liquidationMaxLimit = (uint16((vaultVariables2_ >> 52) & X10) * 10);
+        configs_.withdrawGap = uint16((vaultVariables2_ >> 62) & X10) * 10;
+        configs_.liquidationPenalty = uint16((vaultVariables2_ >> 72) & X10);
+        configs_.borrowFee = uint16((vaultVariables2_ >> 82) & X10);
+    }
+
+    function reduceVaultBorrowLimit(uint256 vaultId) internal {   
+        address vault_ =  VAULT_T1_FACTORY.getVaultAddress(vaultId);
+        address token_ = IFluidVaultT1(vault_).constantsView().borrowToken;
+
+        uint256 userBorrowData_ = LIQUIDITY.readFromStorage(
+            LiquiditySlotsLink.calculateDoubleMappingStorageSlot(
+                LiquiditySlotsLink.LIQUIDITY_USER_BORROW_DOUBLE_MAPPING_SLOT,
+                vault_,
+                token_
+            )
+        );
+
+        uint256 totalBorrowAmount_ = BigMathMinified.fromBigNumber(
+            (userBorrowData_ >> LiquiditySlotsLink.BITS_USER_BORROW_AMOUNT) & X64,
+            DEFAULT_EXPONENT_SIZE,
+            DEFAULT_EXPONENT_MASK
+        );
+        totalBorrowAmount_ = totalBorrowAmount_ * 105 / 100; // 5% increase
+
+        uint256 baseDebtCeiling_ = getRawAmount(token_, 0, 1000, false); // $1000
+
+        AdminModuleStructs.UserBorrowConfig[] memory config_ = new AdminModuleStructs.UserBorrowConfig[](1);
+        config_[0] = AdminModuleStructs.UserBorrowConfig({
+            user: vault_,
+            token: token_,
+            mode: uint8(userBorrowData_ & 1),
+            expandPercent: (userBorrowData_ >>
+                LiquiditySlotsLink.BITS_USER_BORROW_EXPAND_PERCENT) & X14,
+            expandDuration: (userBorrowData_ >>
+                LiquiditySlotsLink.BITS_USER_BORROW_EXPAND_DURATION) & X24,
+            baseDebtCeiling: totalBorrowAmount_ < baseDebtCeiling_ ? totalBorrowAmount_ : baseDebtCeiling_,
```

Comment:
in this case, let's make the original totalBorrowAmount_ with only +0.1% increase as base and the with 5% increase as max? we tried to keep a diff between the two values just out of extra caution